zstacklib/zstacklib/utils/ovn.py

```
@@ -271,7 +271,7 @@ def getVnics(self, brName="br-int"):
             return vnics
         except Exception as err:
             logger.error(
-                "Add port for brdige {} failed. {}".format(brName, err))
+                "Add port for bridge {} failed. {}".format(brName, err))
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)